changelogs/current.yaml

```diff
@@ -118,5 +118,10 @@ new_features:
     Included the asterisk ``*`` in the match pattern when using the * or ** operators in the URL template.
     This behavioral change can be temporarily reverted by setting runtime guard
     ``envoy.reloadable_features.uri_template_match_on_asterisk`` to ``false``.
+- area: tls_inspector filter
+  change: |
+    added :ref:`enable_ja4_fingerprinting
+    <envoy_v3_api_field_extensions.filters.listener.tls_inspector.v3.TlsInspector.enable_ja4_fingerprinting>` to create
+    JA4 fingerprint hash from Client Hello message.
```

Comment:
nit: "to create a JA4 fingerprint hash from the Client Hello message"

Reply:
Done